apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -30,7 +30,7 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
   }
 
   async run() {
-    await this.worker.run<Message>(
+    await this.worker.run<CreateNotificationDto>( // Message type renamed
```

Comment:
Is this comment needed?
I don't understand what it's trying to convey :sweat_smile:

Reply:
nope 🧹